### PR TITLE
A3389

### PR DESCRIPTION
**Pull request by morganvenable**
3389 addition to folders and ci cd

### CODE REVIEW

File: keyboards/svalboard/trackball/pmw3389/rules.mk [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# this turns on trackballs including pimoroni
+POINTING_DEVICE_ENABLE = yes
+# this selects trackball driver - moved to left/right for 3360 default, and 3389 subfolders for 3389
+POINTING_DEVICE_DRIVER = pmw3389   
+# pimoroni_trackball
+# pmw3389
+MH_AUTO_BUTTONS = yes
+
+ifeq ($(strip $(POINTING_DEVICE_DRIVER)), pimoroni_trackball)
```
Comment (ilc) [inline keyboards/svalboard/trackball/pmw3389/rules.mk]: We should remove this stuff.  pimoroni / the other sensor, from both rules.mk.

Minor cleanup.  If urgent, merge, and can be addressed later.

**Review by ilc — COMMENTED** [review]
Overall good, rules.mk could use cleaning up in the 3360 and 3389 directories, but that work can be deferred if merging is urgent.

**Comment by morganvenable** [comment]
@ilc fixed makefiles

**Review by ilc — APPROVED** [review]
LGTM

**Pull request merged**